nextcloud-startup-script.sh

```diff
@@ -377,7 +377,8 @@ whiptail --title "Extra configurations" --checklist --separate-output "Choose wh
 "Security" "(Add extra security based on this http://goo.gl/gEJHi7)" OFF \
 "Static IP" "(Set static IP in Ubuntu with netplan.io)" OFF \
 "Automatic updates" "(Automatically update your server every week on Sundays)" OFF \
-"CookieLifetime" "(Configure forced logout timeout for users using the web GUI)" OFF 2>results
+"CookieLifetime" "(Configure forced logout timeout for users using the web GUI)" OFF \
+"Share-folder" "(If configured, all Nextcloud-shares from other users will be stored in the 'Shared' folder)" OFF 2>results
 
 while read -r -u 9 choice
```

Comment:
no need to change to 10 here?

Reply:
I guess no: but actually I don't know what that number does.

Reply:
Me neither to be honest. :)

Reply:
I saw you changed it in other places though, hence the question.

Reply:
I changed it because I thought it was the max number of options to choose from ;)

Reply:
Maybe we should test if removing the 9 changes anything :D